src/Workspaces/Core/Portable/Options/SerializableOptionSet.cs

```diff
@@ -386,13 +389,12 @@ static bool TryDeserializeOptionKey(ObjectReader reader, ILookup<string, IOption
                     if (option.Feature == feature &&
                         option.IsPerLanguage == isPerLanguage)
                     {
-                        deserializedOptionKey = new OptionKey(option, language);
-                        return true;
+                        return new OptionKey(option, language);
                     }
                 }
 
-                deserializedOptionKey = default;
-                return false;
+                Debug.Fail($"Failed to deserialize: {name}-{feature}-{isPerLanguage}-{language}");
```

Comment:
uncertain if we should be doing this.  is this normal that we aren't able to deserialize an option?  or is this exceptional.

Reply:
Not sure. Is there a way to find out what option failed for the previous case?

Reply:
I can try to debug after applying this fix to find out

Reply:
thanks.  that works for me.